### PR TITLE
Fixed code and unit tests

### PR DESCRIPTION
**Pull request by jbigman**
- fixed search: more section title is not always empty, it can contain "Apps"
- fixed missing comments (ds:9 => ds:8)
- new url called to test throttle,
- updated expected results

### CODE REVIEW

**Comment by jbigman** [comment]
I checked tests with node version 14, 16 and 18 on my windows computer. All of them works. I don't understand why it fails on github.

**Comment by jbigman** [comment]
@facundoolano Please, review this PR to merge other PRs in progress

File: test/lib.search.js [hunk 1]
```diff
@@ -64,23 +64,25 @@ describe('Search method', () => {
...
           assert.equal(apps[0].appId, 'com.netflix.mediaclient');
-          assertIdsInArray(apps, 'com.netflix.ninja', 'com.netflix.NGP.StrangerThings');
+          assertIdsInArray(apps, 'com.netflix.ninja', 'com.netflix.NGP.Poinpy');
         });
     });
 
     it('should return few netflix apps from german store with german language', () => {
       return gplay.search({ term: 'netflix', lang: 'de', country: 'DE' })
         .then((apps) => {
+          console.log(apps.map((app) => app.appId));
```
Comment (facundoolano) [inline test/lib.search.js:79]: Same

**Comment by jbigman** [comment]
I did a re-run on node 14 build and it succeeded : https://github.com/jbigman/google-play-scraper/actions/runs/5686041521

**Comment by jbigman** [comment]
I don't know why the throttle tests timeout here, the build successful passed on my fork. 
https://github.com/jbigman/google-play-scraper/actions/runs/5692222528

**Comment by facundoolano** [comment]
Seems to fail consistently... I'll merge this anyway as it's an improvement from the current code, but we should try to figure out why it doesn't pass.

**Pull request merged**